docs/news/2020-07-06-Dockstore-1.9.0.rst

```diff
@@ -0,0 +1,41 @@
+Dockstore 1.9.0
+===============
+
+Highlighted new features include:
+---------------------------------
+
+-  Preview feature for Galaxy workflow support
+-  Performance improvements for a large variety of endpoints
+-  GitHub app support in preview to allow for automated update of workflow content
+-  Implementation of the `TRS v2 final standard <https://github.com/ga4gh/tool-registry-service-schemas/releases/tag/2.0.0>`_
+
+   -  TRS v2 beta standard support is deprecated, but still present
+
+-  Capture of file and docker image checksums from GitHub, quay.io, Gitlab, and Docker Hub on workflow snapshot to support immutable workflows (see breakdown for details here_)
+-  Support for linking ORCID profiles user profiles
+
+   -  Currently displayed in organization and stargazer views
+
+-  A large number of usability improvements and fixes to the user interface website
```

Comment:
"...to the user interface website" doesn't sound right; maybe:

* ...to the website's user interface
* ...to the website
* ...to the user interface